packs/feat-effects/effect-starlit-transformation.json

```diff
@@ -75,9 +74,7 @@
                     "bolts-of-starlight"
                 ],
                 "property": "damage-type",
-                "selectors": [
-                    "{item|flags.pf2e.rulesSelections.weapon}-damage"
-                ],
+                "selectors": "{item|flags.pf2e.rulesSelections.weapon}-damage",
```

Comment:
In the case of Damage Alteration, you do need an array even if it's a single one

Reply:
made the wrong change...